app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from .errors import (NotVaccinatedError,
+                     OutdatedVaccineError,
+                     NotWearingMaskError)
```

Comment:
Read this message please
https://mate.academy/chats/244801?selectedMessageId=3129519